Taskfile.yml

```diff
@@ -9,21 +9,44 @@ vars:
   DOCKERCMD_SERVER: 'itkdev-docker-compose-server'
 
 tasks:
-  dist:clone:
-    summary: Clone freescout distribution.
+  dist:build:
+    summary: Install Freescout distribution
     cmds:
-      - git clone https://github.com/freescout-help-desk/freescout {{ .TARGET_DIR }}
+      - sh -c "wget https://github.com/freescout-help-desk/freescout/archive/refs/tags/{{.VERSION}}.tar.gz"
+      - sh -c "mkdir freescout"
+      - sh -c "tar -xzvf ./{{.VERSION}}.tar.gz"
+      - sh -c "mv freescout-{{.VERSION}}/* freescout"
+      - sh -c "rm -Rf freescout-{{.VERSION}} {{.VERSION}}.tar.gz"
 
   dev:start-docker:
     cmds:
       - sh -c "{{ .DOCKERCMD }} up --detach --force-recreate --remove-orphans"
 
+  dev:install:
+    - sh -c "cp templates/.env.example freescout/.env"
+    - sh -c "{{ .DOCKERCMD }} exec phpfpm php artisan key:generate"
+
+  dev:install-modules:
+    - sh -c "mkdir freescout/vendor/natxet/cssmin/src"
+    - git clone git@github.com:itk-dev/freescout-itk-issue-create.git {{ .PLUGIN_TARGET_DIR }}/ItkIssueCreate
```

Comment:
the clone modules can be moved into own task and call in both install task... to ensure that both server and dev has the same modules.

Reply:
Don't we clone with https on server and ssh on local dev?

Reply:
We always clone with https (if they are not private repos, then ssh) but the same protocol in both places

Reply:
Changed